TaskEngine/Tasks/QueueAwakerTask.cs

```diff
@@ -269,6 +272,7 @@ protected async override Task OnConsume(JObject jObject, TaskParameters taskPara
                     registerTask(cleanup, "PeriodicCheck");
                     _updateBoxTokenTask.Publish("");
                     _buildElasticIndexTask.Publish("");
+                    _cleanUpElasticIndexTask.Publish("");
```

Comment:
Note: These wont happen in order (even the code suggests they might). They will start at about the same time.

Reply:
It's possible that a recently retired index would not be immediately removed. If it happens, that index would be removed in the next cycle, if it's time-to-live has passed. I think it should not be a problem.